string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,26 @@
+/* exported capitalize */
+
+/* - storage space for the changed word
+   - for each letter in the word
+        - make sure first letter is capitalized
+              - if first letter is not capitalized
+                we capitalize it
+        - if letter is capitalized but is not the
+          first letter, uncapitalize aforementioned
+          letter, and add that letter to the storage
+          space for the changed word
+  - give back capitalized word
+*/
+
+function capitalize(word) {
+  var capitalizedString = '';
+
+  for (var i = 0; i < word.length; i++) {
+    if (i === 0) {
+      capitalizedString = word[0].toUpperCase();
```

Comment:
You can just do this before the loop and then start the loop at `1` to avoid needing this conditional within the loop 🤓